lib/tasks/deployment/20201005203539_populate_case_contact_with_defaults.rake

```diff
@@ -0,0 +1,50 @@
+namespace :after_party do
+  desc 'Deployment task: populate_case_contact_with_defaults'
+  task populate_case_contact_with_defaults: :environment do
+    puts "Running deploy task 'populate_case_contact_with_defaults'"
+
+    CasaOrg.all.each do |org|
+      ContactTypeGroup.find_by(casa_org: org, name: "Biological family").update(name: "Family")
+      
+      ContactTypeGroup.find_by(casa_org: org, name: "Family").tap do |group|
+        ContactType.find_by(contact_type_group: group, name: "Bio Parent")&.destroy
+        ContactType.find_by(contact_type_group: group, name: "Other Family")&.destroy
+
+        ContactType.create(contact_type_group: group, name: "Parent")
+        ContactType.create(contact_type_group: group, name: "Other Family")
+        ContactType.create(contact_type_group: group, name: "Sibling")
+        ContactType.create(contact_type_group: group, name: "Grandparent")
+        ContactType.create(contact_type_group: group, name: "Aunt Uncle or Cousin")
+        ContactType.create(contact_type_group: group, name: "Fictive Kin")
+      end
+
+      ContactTypeGroup.find_by(casa_org: org, name: "Placement").tap do |group|
+        ContactType.create(contact_type_group: group, name: "Caregiver Family")
+        ContactType.create(contact_type_group: group, name: "Therapeutic Agency Worker")
+      end
+
+      ContactTypeGroup.find_by(casa_org: org, name: "Social Services").tap do |group|
+        ContactType.find_by(contact_type_group: group, name: "DSS Worker").destroy
+        ContactType.create(contact_type_group: group, name: "Youth")
+        ContactType.create(contact_type_group: group, name: "Supervisor")
+      end
+
+      ContactTypeGroup.find_by(casa_org: org, name: "Health").tap do |group|
+        ContactType.find_by(contact_type_group: group, name: "Therapist").update(name: "Mental Health Therapist")
+        ContactType.create(contact_type_group: group, name: "Other Therapist")
+        ContactType.create(contact_type_group: group, name: "Psychiatric Practitioner")
+      end
+
+      ContactTypeGroup.find_by(casa_org: org, name: "Education").tap do |group|
+        ContactType.create(contact_type_group: group, name: "Guidance Counselor")
+        ContactType.create(contact_type_group: group, name: "Teacher")
+        ContactType.create(contact_type_group: group, name: "IEP Team")
+      end
```

Comment:
Can you just combine this task with `populate_contact_type_groups_and_contact_types` and create these all in one task?